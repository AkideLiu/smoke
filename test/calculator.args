fixtures/calculator.rb
fixtures/calculator
