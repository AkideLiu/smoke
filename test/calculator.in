--separated-args fixtures/calculator fixtures/calculator.rb
