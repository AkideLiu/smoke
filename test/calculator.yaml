tests:
  calculator:
    args:
      - fixtures/calculator
    exit-status: 1
    stdout:
      file: io/calculator.out
