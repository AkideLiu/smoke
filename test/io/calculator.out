addition
  succeeded
subtraction
  succeeded
multiplication
  succeeded
division
  succeeded
three-numbers
  input:  5 + 7 + 12
  output: @@ -1 +1 @@
          -24
          +12
parentheses
  input:  (6 + 3) * 2
  output: @@ -1 +1 @@
          -18
          +3
complex
  input:  (8 + 2i) + (7 - 4i)
  output: @@ -1 +1 @@
          -15 - 2i
          +2
infix-words-error
  succeeded

8 tests, 3 failures
